### PR TITLE
feat: check if repository exists

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #5 
Added endpoint to handle repository head request, extracted method in `RepoSettings` to get repo settings key, added default `RequestAttr.readOrThrow()` method to obtain required request attribute or throw exception.

### CODE REVIEW

File: src/main/java/com/artipie/front/api/HeadRepository.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        this.stn = stn;
+    }
+
+    @Override
+    public String handle(final Request request, final Response response) {
+        this.stn.key(
+            request.params(GetRepository.PARAM),
+            RequestAttr.Standard.USER_ID.readOrThrow(request)
+        );
+        return "OK";
```
Comment (g4s8) [inline src/main/java/com/artipie/front/api/HeadRepository.java:38]: For `HEAD` requests we don't need to return any body

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova just one comment

**Comment by olenagerasimova** [comment]
@g4s8 thanks, corrected

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**